uniformjs-rails.gemspec

```diff
@@ -12,5 +12,5 @@ Gem::Specification.new do |gem|
   gem.name          = "uniformjs-rails"
   gem.require_paths = ["lib"]
   gem.version       = Uniformjs::Rails::VERSION
-  gem.add_dependency "railties", "~> 3.1"
+  gem.add_dependency "railties", "~> 4.1"
```

Comment:
It does't seems like we need to depend on a specific version of railties. Could you relax this to say `>= 4.1` so that the gem also works with rails 5?